### PR TITLE
Fix new release message

### PR DESCRIPTION
**Pull request by sebn**

### CODE REVIEW

File: gui/app/main.js [hunk 1]
```diff
@@ -172,17 +172,17 @@ const checkForNewRelease = () => {
...
     newReleaseAvailable = true
-    sendToMainWindow('new-release-available', releaseNotes || '', releaseName || '')
+    sendToMainWindow('new-release-available', releaseNotes, releaseName)
   })
   autoUpdater.addListener('error', (err) => console.error(err))
   autoUpdater.checkForUpdates()
   setInterval(() => {
     autoUpdater.checkForUpdates()
-  }, 86400) // Check if a new release is available once per day
+  }, 1000 * 60 * 24) // Check if a new release is available once per day
```
Comment (sebn) [inline gui/app/main.js]: Oops, didn't I miss another `* 60`? 😄

File: gui/lib/Settings.elm [hunk 2]
```diff
@@ -76,7 +76,7 @@ view helpers model =
                 Just ( notes, name ) ->
                     [ h2 [] [ text (helpers.t "Settings A new release is available") ]
                     , p [] [ text name ]
-                    , p [] [ text notes ]
+                    , iframe [ srcdoc notes ] []
```
Comment (sebn) [inline gui/lib/Settings.elm]: Forgot to remove it (overkill)

File: gui/app/main.js (hunk 1 above)
Comment (sebn) [inline gui/app/main.js]: Fixed

File: gui/lib/Settings.elm (hunk 2 above)
Comment (sebn) [inline gui/lib/Settings.elm]: Fixed

**Comment by sebn** [comment]
@nono sorry for the noise, should be ready for review now.

File: gui/app/main.js [hunk 3]
```diff
@@ -172,17 +172,17 @@ const checkForNewRelease = () => {
...
     newReleaseAvailable = true
-    sendToMainWindow('new-release-available', releaseNotes || '', releaseName || '')
+    sendToMainWindow('new-release-available', releaseNotes, releaseName)
   })
   autoUpdater.addListener('error', (err) => console.error(err))
   autoUpdater.checkForUpdates()
   setInterval(() => {
     autoUpdater.checkForUpdates()
-  }, 86400) // Check if a new release is available once per day
+  }, 1000 * 60 * 60 * 24) // Check if a new release is available once per day
```
Comment (nono) [inline gui/app/main.js]: Why `1000`?

Comment (sebn) [inline gui/app/main.js]: milliseconds?

Comment (nono) [inline gui/app/main.js]: Oh! It was bugged before (seconds instead of milliseconds)

Comment (sebn) [inline gui/app/main.js]: yep 72031cc8b38d7eb5169ac64b8539299178d95e48 😃

**Pull request merged**